docs/podman-rm.1.md

```diff
@@ -9,7 +9,9 @@ podman\-container\-rm (podman\-rm) - Remove one or more containers
 **podman rm** [*options*] *container*
 
 ## DESCRIPTION
-**podman rm** will remove one or more containers from the host.  The container name or ID can be used.  This does not remove images.  Running containers will not be removed without the `-f` option
+**podman rm** will remove one or more containers from the host.  The container name or ID can be used.  This does not remove images.
+Running containers will not be removed without the `-f` option.
+Containers with an invalid runtime configuration can be forcefully removed (i.e., evicted) with the `-ff` option.
```

Comment:
I would prefer --evict vs -ff

Reply:
Oh @jwhhonce, that's a capital thougth.  +1000

Reply:
I concur with that. The semantics are different so I find `-ff` confusing (as it's not an extension of doing more but just something different).

Reply:
`-ff` is still there. Has there been a change of plans to keep it?